File: api/code-samples/python/debug/profiling/setBlockProfileRate.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

rate = 4

sdk = OpenSDK(host)
debug_response = sdk.debug.set_block_profile_rate(rate)

print(json.loads(debug_response.response.data))

File: web3rpc/rpc-specs/code-samples/javascript/debug/profilling/setBlockProfileRate.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
  const rate = 3;

  provider.debug
    .setBlockProfileRate(rate, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
